app/components/primer/button_component.rb

```diff
@@ -96,6 +111,12 @@ class ButtonComponent < Primer::Component
     #     Button
     #   <% end %>
     #
+    # @example With tooltip
+    #   <%= render(Primer::ButtonComponent.new(id: "button-with-tooltip")) do |c| %>
+    #     <% c.tooltip(text: "Tooltip text") %>
+    #     Button
+    #   <% end %>
```

Comment:
Maybe this is overkill, but should we link to `Tooltip` documentation here too using `@code` and `@description`?

Reply:
Not overkill at all, good idea!